### PR TITLE
wip: bump docker images versioning

### PR DESCRIPTION
**Pull request by Kodylow**

### CODE REVIEW

File: docker/fedimintd-mutinynet/docker-compose.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ version: "3"
 
 services:
   fedimintd:
-    image: fedimint/fedimintd:v0.1.3
+    image: fedimint/fedimintd:v0.2.0-rc5
```
Comment (douglaz) [inline docker/fedimintd-mutinynet/docker-compose.yaml:6]: 9?

File: docker/gateway-mutinynet/docker-compose.yaml [hunk 2]
```diff
@@ -84,11 +85,12 @@ services:
     restart: always
 
   thunderhub:
-    image: apotdevin/thunderhub:base-v0.13.20
+    image: local/fedimintui-thunderhub
```
Comment (douglaz) [inline docker/gateway-mutinynet/docker-compose.yaml:88]: :thinking:

File: gateway/ln-gateway/src/lib.rs [hunk 3]
```diff
@@ -481,6 +481,7 @@ impl Gateway {
                                                 warn!("Lightning node does not match previously configured gateway network : ({:?})", config.network);
                                                 info!("Changing gateway network to match lightning node network : ({:?})", lightning_network);
                                                 self.handle_disconnect(htlc_task_group).await;
+                                                self.set_gateway_state(GatewayState::Connected).await;
```
Comment (douglaz) [inline gateway/ln-gateway/src/lib.rs:484]: :thinking:

**Comment by Kodylow** [comment]
@douglaz we can close this, this was back when I was debugging the gateway not connecting to thunderhub / voltage . Will open a new one when we cut the 0.2 release.

**Comment by elsirion** [comment]
Ping we got a 0.2.0 release :tada:

**Pull request closed without merging**